opennms-base-assembly/src/main/filtered/etc/resource-types.d/jmx-resource.xml

```diff
@@ -16,4 +17,35 @@
       </storageStrategy>
     </resourceType>
 
+    <!-- Netflow 5 -->
+    <resourceType name="netflow5" label="Netflow 5" resourceLabel="${index}">
```

Comment:
We shouldn't need to define a different resource type for every protocol - we should be able to model this using a single resource type using the index to represent the different types.

Reply:
Done